crates/re_components/tests/data_table.rs

```diff
@@ -34,15 +34,16 @@ fn data_table_sizes_basics() {
         BooleanArray::from(vec![Some(true), Some(false), Some(true)]).boxed(),
     );
     cell.compute_size_bytes();
+    // TODO(jleibs): Understand this size difference
     expect(
         cell.clone(), //
         10_000,       // num_rows
-        2_690_064,    // expected_num_bytes
+        2_770_072,    // expected_num_bytes
```

Comment:
@teh-cmc seems like the change of ComponentName from interned-string to Cow increased the sizes? Thoughts on whether this seems reasonable or if we should be taking a different approach here?